usecases/optimization_paper/README.md

```diff
@@ -0,0 +1,38 @@
+## Overview
+This directory contains to code to automatically generate the planned paper.
+This includes the scripts to generate the figures and at least the option to re-run the computations if needed.
+
+## Structure
+The current proposal is the write the paper in the **tex** folder and split the document in sections, which are located in **tex/tex_sections**.
+Modules like **workflow_graph** can be added as directory to the main directory and the included in the pydoit workflow.
+Figures generated for the paper are located in **figures**.
+
+## Features
+
+### pydoit
+There is a **dodo.py** file which includes all steps required to generate the paper.
+Expensive computations should not be always. We will need to find a good solution.
+I would propose to use a similar setup as in the MinimumWorkingExample and us a call like 
+```
+doit mode=full
```

Comment:
I would rather propose it the other way round, if we are interested in a subset to exclude something, I would add a ci option

Reply:
I will setup the possibility to implement different options, we can then always decide how to use them.